test/runtests.jl

```diff
@@ -4,6 +4,6 @@ using Images
 using LaTeXStrings
 using Test, ReferenceTests
```

Comment:
According to BlueStyle, packages should each be on their own line and each `using` should be in alphabetical order.